pages/sigmaker.tsx

```diff
@@ -150,40 +159,31 @@ const Sigmaker: NextPage = () => {
         <br />
         <StyledGrid container spacing={2}>
           <Grid item xs={12} md={6}>
-            <Typography variant="h5"> Enter your info here! </Typography>
-            <StyledFormControl>
-              <Stack spacing={3}>
-                {createInputField('fullName', formData.fullName, 'Full name')}
-                {createInputField('title', formData.title, 'Title')}
-                {createInputField('phone', formData.phone, 'Phone')}
-                {createInputField(
-                  'email',
-                  formData.email,
-                  'Email (@hackbeanpot.com)'
-                )}
-              </Stack>
-            </StyledFormControl>
-            <div>
+          <ValidatorForm
+                onSubmit={handleSubmit}
+            >
+            <Stack spacing={3}>
+              <Typography variant="h5"> Enter your info here! </Typography>
+              {createValidatedInputField('fullName', formData.fullName, 'Full name')}
+              {createValidatedInputField('title', formData.title, 'Title')}
+              {createValidatedInputField('phone', formData.phone, 'Phone')}
+              {createValidatedInputField('email', formData.email, 'Email (@hackbeanpot.com)')}
               <StyledButton
                 size="large"
                 color="info"
                 variant="contained"
-                width="medium"
-                onClick={() => {
-                  setSignatureData(formData)
-                  setFormData({
-                    fullName: '',
-                    title: '',
-                    phone: '',
-                    email: ''
-                  })
-                }}
+                type="submit"
+                sx={{width:220}}
```

Comment:
We don't want to use inline styling - to make this button wider you can use width="medium"